README.md

```diff
@@ -485,6 +485,7 @@ Services to securely store your Docker images.
 
 # Useful Resources
 * [Awesome Linux Container](https://github.com/Friz-zy/awesome-linux-containers) more general about container than this repo, by [@Friz-zy](https://github.com/Friz-zy).
+* [Awesome Selfhosted](https://github.com/Kickball/awesome-selfhosted) list of Free Software network services and web applications which can be hosted locally by [Kickball](https://github.com/Kickball)
```

Comment:
@devhkr 
Great idea!!!
I'm away until Sunday with no access to a computer. Can you add a commit to this pr with the changes you mentioned?